src/Platform/Microsoft.Testing.Extensions.VSTestBridge/ObjectModel/FastFilter.cs

```diff
@@ -46,9 +46,17 @@ internal bool Evaluate(Func<string, object?> propertyValueProvider)
         bool matched = false;
         foreach (string name in FilterProperties.Keys)
```

Comment:
Looks like this whole implementation is a copy from vstest.

1. Is there no public API from VSTest that we can call here?
2. I think we should get those changes first in vstest repo, test it there, and then replicate here in testfx.